examples/gwlb_with_vmseries/versions.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = ">= 1.5, < 2.0"
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+    random = {
+      source = "hashicorp/random"
+    }
+    http = {
+      source = "hashicorp/http"
+    }
```

Comment:
I think we don't use this in this example?

Reply:
Updated